packages/wonder-blocks-button/components/button-core.js

```diff
@@ -0,0 +1,271 @@
+// @flow
+import React from "react";
+import {StyleSheet, css} from "aphrodite";
+
+import {LabelLarge} from "wonder-blocks-typography";
+import {processStyleList} from "../../wonder-blocks-core/util/util.js";
+import Color, {mix, fade} from "wonder-blocks-color";
+import type {ValidTints} from "wonder-blocks-color";
+import type {SharedProps} from "../types.js";
+
+type Props = SharedProps & {
+    hovered?: boolean,
+    focused?: boolean,
+    pressed?: boolean,
+};
+
+export default class ButtonCore extends React.Component<Props> {
+    render() {
+        const {
+            color,
+            kind,
+            light,
+            size,
+            disabled,
+            hovered,
+            focused,
+            pressed,
+        } = this.props;
+
+        const buttonStyles = _generateStyles(
+            color || Color.blue,
```

Comment:
Ah, I think it would be better to change this so that we have the following in the class, then we can just pass them in here and it'll work!

```
static defaultProps = {
    color: Color.blue,
    kind: "primary",
    light: false,
};
```

Reply:
We could also add `state: "default"` as part of the defaultProps.